pkg/apis/kubekey/v1alpha1/plugins_types.go

```diff
@@ -2,17 +2,21 @@ package v1alpha1
 
 type Storage struct {
 	DefaultStorageClass string      `yaml:"defaultStorageClass" json:"defaultStorageClass,omitempty"`
-	LocalVolume         LocalVolume `yaml:"localVolume" json:"localVolume,omitempty"`
-	NfsClient           NfsClient   `yaml:"nfsClient" json:"nfsClient,omitempty"`
-	GlusterFS           GlusterFS   `yaml:"glusterFS" json:"glusterFS,omitempty"`
-	CephRBD             CephRBD     `yaml:"cephRBD" json:"cephRBD,omitempty"`
+	LocalVolume         LocalVolume `yaml:"local" json:"locaVolume,omitempty"`
+	NfsClient           NfsClient   `yaml:"nfs" json:"nfs,omitempty"`
```

Comment:
why here is nfsclient, while the yaml is nfs?

Reply:
The name of this plugin is nfsclient.

Reply:
[nfs](https://github.com/kubernetes-incubator/external-storage/tree/master/nfs) and [nfs-client](https://github.com/kubernetes-incubator/external-storage/tree/master/nfs-client) are two different **external-storage** of [kubernetes-incubator](https://github.com/kubernetes-incubator/external-storage) .  Actually what we use is nfs-client, i suggest use nfs-client for unity and no confoud.

Reply:
Yes. Then we'll still use `nfsClient`. @rayzhou2017 Does it need to be changed in readme？

Reply:
yes, if it is that case, then make it consistent.